vermin/source_visitor.py

```diff
@@ -1217,6 +1236,13 @@ def visit_Constant(self, node):
       for directive in BYTES_DIRECTIVE_REGEX.findall(str(node.value)):
         self.__add_bytes_directive(directive, node.lineno)
 
+    # From 3.8, Ellipsis() is represented as Constant(value=Ellipsis)
+    if is_ellipsis_node(node):
+      # Force identity testing instead of equality testing.
+      if not any(n is node for n in self.__ellipsis_nodes_in_slices):
+        self.__ellipsis_out_of_slices = True
+        self.__vvprint("ellipsis literal (`...`) out of slices", versions=[None, (3, 0)])
```

Comment:
These lines are the same as 1914-1917. Maybe reuse in another function or call `self.visit_Ellipsis(node)`.